include/debug.h

```diff
@@ -42,6 +45,9 @@ noreturn void panic(const char *fmt, ...) __printf(1, 2);
 #define error(...) log(LOG_STRING_ERROR __VA_ARGS__)
 #define warn(...)  log(LOG_STRING_WARNING __VA_ARGS__)
 #define info(...)  log(LOG_STRING_INFO __VA_ARGS__)
+#if TEST
+#define test(...) log(LOG_STRING_DEBUG __VA_ARGS__)
```

Comment:
You need to define a `LOG_STRING_TEST` above with the same number as the order in the list (`\x06`)

Reply:
done